data/agent_attributes.yaml

```
@@ -53,6 +53,12 @@ attributes:
   required: false
   desc: |
       The number of idle seconds to wait before shutting down. When setting this option alongside `spawn`, all agents started by `spawn` must be idle before the timeout counter will begin. After the timeout, the spawned agents will all disconnect.
+- name: enable-job-log-tmpfile
+  env_var: BUILDKITE_ENABLE_JOB_LOG_TMPFILE
+  default_value: "false"
+  required: false
+  desc: |
+      Enables the writing of job logs into a temporary file which can be accessed via the `BUILDKITE_JOB_LOG_TMPFILE` environment variable. Must be using Agent v3.36.0 or above.
```

```suggestion
      Enables the writing of job logs into a temporary file which can be accessed via the `BUILDKITE_JOB_LOG_TMPFILE` environment variable. Introduced in [v3.36](https://github.com/buildkite/agent/releases/tag/v3.36.0).
```

Could possibly update this wording to resemble the existing wording [here](https://buildkite.com/docs/agent/v3/configuration#redacted-vars:~:text=Introduced%20in%20v3.31.)